ra-tls/stub-enclave/Wolfssl_Enclave.c

```diff
@@ -6,6 +6,10 @@
 
 #include "sgx_trts.h"
 
+#include "wolfssl-ra-attester.h"
+#include "../libenclave-tls/ra-common.h"
+
+#define EPID "epid"
```

Comment:
You define this quote type several places. It should be a common thing and part of enclave tls layer.